types/svg-render/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for svg-render 1.2
+// Project: https://github.com/catdad-experiments/svg-render
+// Definitions by: Sergey Chernov <https://github.com/sergeychernov/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace svgRender {
+  export interface SvgRenderOptions {
+      buffer: Buffer;
+      width?: number;
+      height?: number;
+      expandUseTags?: boolean;
+  }
+  function svgRender(options: SvgRenderOptions): Buffer;
```

Comment:
This should go outside the `declare namespace` block—I might have made a typo in my suggestion.